spec/active_record_query_counter_spec.rb

```diff
@@ -184,6 +187,16 @@
         end
       end
     end
+
+    it "keeps a count of rollbacks for each transaction trace" do
+      ActiveRecordQueryCounter.count_queries do
+        TestModel.transaction do
```

Comment:
You can you add some more steps to this test to perform multiple transactions.

- Add a successful transaction and assert that the rollback count is still 1
- Add another rollback and assert that the rollback count has incremented again to 2